.github/workflows/yaml-lint.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: YAML lint
+
+on:
+  pull_request:
+    paths:
+      - "**.yaml"
+      - "**.yml"
+
+jobs:
+  yamllint:
+    name: yamllint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v4.0.0
```

Comment:
Could you also add dependabot to be updating these @ versions?

Reply:
PR is in for that.